.travis.yml

```diff
@@ -1,31 +1,37 @@
 language: php
-sudo: false
-
-php:
-  - 7.0
-  - 5.6
-  - 5.5
-  - 5.4
-  - hhvm
-  - hhvm-nightly
 
 matrix:
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS='--prefer-lowest'
```

Comment:
Would it be simpler to just add an env block later? e.g.
```
env:
  matrix:
    - COMPOSER_FLAGS="--prefer-lowest"
    - COMPOSER_FLAGS=""
```
Keeps the list of flags and versions distinct?